File: go/Begin/task_01/main.go
package main

import (
	"fmt"
)

func main() {
	var a float64

	fmt.Print("Enter the side of the square: ")
	fmt.Scan(&a)
	
	p := 4 * a
	fmt.Printf("Perimeter of the square: %.2f\n", p)
}

File: java/Begin/task_01/main.java
import java.util.Scanner;

public class main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the side of the square: ");
        double a = scanner.nextDouble();

        double p = 4 * a;
        System.out.printf("Perimeter of the square: %.2f\n", p);
    }
}